### PR TITLE
Fix easydict feedstock

### PR DESCRIPTION
**Pull request by benglewis**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by benglewis** [comment]
@conda-forge-admin, please rerender

**Comment by benglewis** [comment]
@conda-forge-admin Can we please merge this PR? 🙏

**Comment by benglewis** [comment]
@beckermr Is it possible to merge this? 🙏

**Comment by benglewis** [comment]
@beckermr 
@zhaofeng-shu33
@conda-forge/core

Any chance that this will be merged at some point? 🙏 I don't like pushing, but it has been two weeks and I did not even get any reply yet. I believe that the PR should be safe to merge and fix newer `easydict` versions showing up on `conda-forge`

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-    summary: Access dict values as attributes (works recursively).
+  home: https://github.com/makinacorpus/easydict
+  license: LGPL-3.0-only
+  license_file: LICENSE
+  summary: Access dict values as attributes (works recursively).
 extra:
-    recipe-maintainers:
-        - zhaofeng-shu33
+  recipe-maintainers:
+    - zhaofeng-shu33
```
Comment (hmaarrfk) [inline recipe/meta.yaml:50]: Are you willing to help maintain the recipe long term?

**Comment by hmaarrfk** [comment]
hmm noarch recipes are safer to merge in even without a rerender.

Please submit an issue to add yourself and add a reference to this PR for others to know you have put in your time for this feedstock!

**Pull request merged**